### PR TITLE
getting started

### PR DESCRIPTION
**Pull request by wedodge89**
just starting out with node. not currently functional.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+        aws.config.update({
+            accessKeyId: process.env.ACCESS,
+            secretAccessKey: process.env.SECRET,
+            region: "us-east-1"
+        });
+
+        const s3 = new aws.S3();
+        const response = await s3.listObjectsV2({
+            Bucket: "cru.org-storylines"
```
Comment (wrandall22) [inline index.js]: Should probably make this an environment variable as well.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require("dotenv").config();
+const aws = require("aws-sdk");
+
+(async function() {
+    try {
+        
+        aws.config.setPromisesDependency();
+
+        aws.config.update({
+            accessKeyId: process.env.ACCESS,
```
Comment (wrandall22) [inline index.js:10]: I assume you've already tested (via logs or some other way) that these environment variables are coming through to the code?

Comment (wedodge89) [reply index.js:10]: Yep. They are console logging just fine.

File: index.js (hunk 1 above)
Comment (wedodge89) [reply index.js]: Done.

File: index.js (hunk 2 above)
Comment (wrandall22) [reply index.js:10]: They may need to be added to [environment.js](https://github.com/CruGlobal/cloudsearch-s3-handler/blob/master/serverless/environment.js)

File: package.json [hunk 3]
```diff
@@ -1,52 +1,23 @@
...
-  "scripts": {
-    "lint": "standard --fix --verbose || exit 0",
-    "test": "jest --detectOpenHandles --coverage"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@types/jest": "^24.0.15",
-    "aws-sdk": "^2.470.0",
-    "babel-jest": "^24.9.0",
```
Comment (wrandall22) [inline package.json]: You'll want to keep most of these devDependencies I think.

**Pull request merged**